src/components/common/layout/base.njk

```diff
@@ -150,7 +150,8 @@
       }, {
         disableGa4Tracking: {{isGa4Disabled}},
         disableUaTracking: {{isUaDisabled}},
-        cookieDomain: "{{analyticsCookieDomain}}"
+        cookieDomain: "{{analyticsCookieDomain}}",
+        isDataSensitive: true
```

Comment:
My understanding is that `isDataSensitive` should be `true` by default now, and that the way to disable it (I'm really not sure why we should ever want to disable it – the instructions on the README haven't really expanded on that) is to set `isDataSensitive: false`

Reply:
It is `true` by default but I thought it'd be better here to be explicit and set it ourselves since it's a setting that protects personal data